example/lib/screens/checkout_screen.dart

```diff
@@ -174,7 +174,12 @@ class _CheckoutScreenState extends State<CheckoutScreen> {
       builder: (var context) => _SettingsModalPopup(
         locationName: location.name,
         onReaderSDKSettings: onReaderSDKSettings,
-        onDeauthorize: onDeauthorize,
+        onDeauthorize: () => confirmOnDeauthorize(
```

Comment:
can we keep the callback code style consistent with `onReaderSDKSettings: onReaderSDKSettings,`?